GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_NASA.cfg

```diff
@@ -237,7 +237,7 @@
 	@description = Engine core for the Space Launch System (SLS). Features four Rocketdyne RS-25D/E engines. [8.4 m]
 }
 
-@PART[Size3LargeTank]:FOR[RealismOverhaul]
+@PART[Size3LargeTank]:FOR[RealismOverhaul]:NEEDS[!ReStock]
```

Comment:
What is the intent here?  If ReStock is installed, then this tank will not be RO-configured at all.  If we don't want ReStock to touch Size3LargeTank because we have a better or exc,usive use for it, we should just black-list it for ReStock, instead of worrying about what they modify.

Reply:
when you have restock installed this tank has invisible tanks and no collider to click to pick up.

Reply:
ReStock *also* employs the model-construction technique we use.  So `!MODEL {}` is insufficient to deal with all the MODELs added in <https://github.com/PorktoberRevolution/ReStocked/blob/master/Distribution/Restock/GameData/ReStock/Patches/FuelTank/restock-fueltanks-375.cfg#L128-L150>

Does ReStocked Size3LFB represent an SLS?  If so, we should allow what they do, apply appropriate rescaling, and then set our mass/etc in a separate patch.  If not, we should blacklist them touching this part with:
`@PART[Size3LargeTank]:NEEDS[ReStock]:FIRST { %RestockIgnore = True }`

Reply:
"Does ReStocked Size3LFB represent an SLS?" Not sure.